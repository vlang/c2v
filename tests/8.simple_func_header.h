void testFunc() {

}

void testFunc2(int remove) {

}

void testFunc3(int shared) {

}

void testFunc4(int select) {

}
