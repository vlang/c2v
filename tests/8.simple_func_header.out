[translated]
module tests

fn C.testFunc()

pub fn testfunc() {
	C.testFunc()
}
